generator/templates/schemavalidator.gotmpl

```diff
@@ -247,6 +247,18 @@ func ({{ .ReceiverName }} *{{ if .IsExported }}{{ pascalize .Name}}{{ else }}{{
 // Validate validates this {{ humanize .Name }}
 func ({{.ReceiverName}} {{ if or .IsTuple .IsComplexObject .IsAdditionalProperties }}*{{ end }}{{ if .IsExported }}{{ pascalize .Name}}{{ else }}{{ .Name }}{{ end }}) Validate(formats strfmt.Registry) error {
   var res []error
+
+  {{ if .IsPrimitive }}
+    var propName string
```

Comment:
The use case of this particular issue is I'm using a common modal component (e.g. `type IdentifierType string`), while this common modal (`IdentifierType`) are referenced by many different places of path/modal/property definition through `$ref`. When the `schemavalidator` template is rendered, the `IdentifierType` modal validator (methods) are not able to know what components it's validating for, actually the `IdentifierType` modal is just a common modal for other components. While, in runtime, who's calling the `IdentifierType` is determined and so being able to parsed out. This's mostly based on testing and may also be constrained by the implementation, please update if any concerns.